packages/core/src/modules/oob/helpers.ts

```diff
@@ -0,0 +1,53 @@
+import { AriesFrameworkError } from '../../error'
+import { ConnectionInvitationMessage, HandshakeProtocol } from '../connections'
+import { DidCommService } from '../dids'
+
+import { OutOfBandMessage } from './messages'
+
+export function convertToNewInvitation(oldInvitation: ConnectionInvitationMessage) {
```

Comment:
Yeah, I kind of count with a migration script that would create an OOB record and remove the old invitation from all connections. 

I haven't thought about the possibility of removing OOB and having connections without an OOB record. It could work. There shouldn't be any reason why it wouldn't work without them. On the other hand, it can be helpful in some auditing purposes.

We definitely need to store reusable OOB records from the sender side.